### ISSUE TITLE
Busybox not found

### ISSUE THREAD

**Issue by AdamFM** [DroidPHP/DroidPHP#79]
![error2](https://cloud.githubusercontent.com/assets/5110060/11113403/24360d06-894f-11e5-84f5-1c5c1d0bc4d1.jpg)

how to fix it.. where i can get that busybox?

**Comment by vancentvan** [comment]
There does exist busybox binary file on the internet, you can find it and push it to your system..

**Comment by Duna** [comment]
Yes because you have changed the package name of the app. you have to update with your pacjage name in Constants class.

#93 - is already compiled, but server still not starting
